pkg/registry/lock/registry.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
@wojtek-t, @mikedanese - these semantics appear to be implied, is there a document somewhere?